tasks/main.yml

```diff
@@ -98,3 +98,12 @@
     regexp: '^NEEDRESTART\s+sudo -n -- /usr/sbin/needrestart -p'
     state: absent
   when: not (needrestart_checkmk_mrpe | default(False))
+
+- name: Register custom restart scripts
+  ansible.builtin.template:
+    src: "{{ item.script }}"
```

Comment:
Maybe I'm missing something, but in your examples the `script` attribute contains the script content, not the path. The `src` attribute to the Ansible template module requires a path to file, not the content. Maybe you want to use the `copy` module with the `content` attribute instead?

Reply:
You're absolutely right. I have no idea how I slipped that in.
Maybe because it was already too late in the evening ^^